src/concepts/kafka-proxies/mqtt-proxy.md

```diff
@@ -70,7 +70,11 @@ An MQTT client can Publish messages to any configured Kafka topics, marking spec
 
 ### Session Management
 
-MQTT connect, disconnect, and other session messages are maintained on the `sessions` Kafka topic.
+MQTT connect, disconnect, and other session messages are maintained on the the log compacted `sessions` Kafka topic. A message keyed by the MQTT client ID on the topic is used to track client subscriptions across client reconnects.
```

Comment:
`sessions` or `mqtt-sessions`?

Reply:
it should be `sessions` since that is the yaml keyword. but It would be good to add a link to the reference section.